python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -75,6 +87,21 @@ class AssistantAgentConfig(BaseModel):
     model_client_stream: bool = False
     reflect_on_tool_use: bool
     tool_call_summary_format: str
+    tool_call_loop: bool = False
```

Comment:
Can we have only one parameter `tool_call_loop_config: ToolCallLoopConfig | None = None`, and when it is None it matches the current tool call behavior?